csharp/ql/src/utils/model-generator/internal/CaptureSummaryFlow.qll

```diff
@@ -79,3 +79,16 @@ string captureFlow(TargetApi api) {
   result = captureQualifierFlow(api) or
   result = captureThroughFlow(api)
 }
+
+/**
+ * Gets the negative summary for `api`, if any.
+ * A negative summary is generated, if there does not exist any positive flow that
+ * shares the same summary prefix - otherwise these models will be indistinguishable.
+ */
+string captureNoFlow(TargetApi api) {
+  not exists(TargetApi other, string flow |
+    flow = captureFlow(other) and
+    partialNegativeModel(other) = partialNegativeModel(api)
```

Comment:
Does this join perform well enough? It looks suspicious.

Reply:
Hmm, is `partialNegativeModel` a bijection? I assume not, since it's used here, but I can't quite see why.

Reply:
Good catch. This was needed in one of previous versions of the Negative summaries (the syntax and information level has changed since this was introduced). I will remove it.

Reply:
It turns out when generating models for .NET runtime, the source of the reference assemblies interferes with the model generation. Eg. we have
```csharp
System.Linq.Expressions.Expression System.Linq.Expressions.IArgumentProvider.GetArgument(int index) { throw null; }
public Expression GetArgument(int index) => _arguments[index];
```
Which means that both a positve and negative flow summary is generated.